docs/01-programs/01-token-metadata/03-instructions.md

```diff
@@ -343,34 +343,95 @@ Burns an asset, closing associated accounts.
+| `Collection` delegate                                     | `UpdateArgs::CollectionDelegateV2`                    |`collection` for self and any children assets if this is a collection parent |
+| `ProgrammableConfigItem` delegate                         | `UpdateArgs::ProgConfigItemDelegateV2`                |`rule_set` for self only |
+| `ProgrammableConfig` delegate                             | `UpdateArgs::V1`*, `UpdateArgs::ProgConfigDelegateV2` |`rule_set` for self and any children assets if this is a collection parent |
+| Token holder                                              | Not currently supported                               | Not applicable |
+| All token delegates (`TokenDelegateRole::Transfer`, etc.) | None                                                  | Not applicable |
+
+_*For backwards compatibility._
+
+#### Specific limitations
+- Creators and collections cannot be set to verified by this instruction if they already in the asset's metadata as unverified.  Conversely, this instruction cannot unverify creators or collections if they are already in the asset's metadata as verified.
+- `primary_sale_happened` can only be switched from `False` to `True` and cannot be switched back.
+- `is_mutable` can only be switched from `True` to `False` and cannot be switched back.
+- If the token standard is inferred to be or already set to `Fungible` or `FungibleAsset`, the item update authority or the `MetadataDelegateRole::Authority` can freely switch the asset between those two standard types (see below for more on token standard inference).
+- `collection_details` can only be used to set size on an unsized collection, and only once.  Once the collection size is set it is managed by the token-metadata program when items are added/removed from the collection using other instructions (i.e., `Verify`, `Burn`).
+
+#### Token standard inference
+If the asset's token standard is unknown (because it was created with legacy instructions that did not set it), then this handler will infer and set the appropriate standard, based on whether the asset has an edition account as the mint authority, the type of the edition account, the current supply, and decimals.
+
+If the asset's mint authority is a master edition account, but the master edition account was not provided to the `Update` handler, the handler will detect this and fail rather than erroneously inferring the asset is a `FungibleAsset`.
+
+#### Specifying optional accounts
+- The `delegate_record` optional account is only required if using a delegate.
+- The `token` optional account is required if the `RuleSet` is being changed.  This is because we do not allow for a `RuleSet` to be changed if the token currently has a delegate.  The `token` account is also needed if the authority is the owner/holder.  Also, if updates byt the owner/holder were supported (currently they are not), then the `token` account would be needed for authorizing the owner/holder.
```

Comment:
`if updates byt` --> `if update by`

Reply:
Done also reworded to make overall statements clearer.